### PR TITLE
Update release notes for 1.3.0 release

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
@TomSweeneyRedHat @baude @vrothberg @rhatdan  PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+- The `podman runlabel` command now allows a `$GLOBAL_OPTS` variable, which will be populated by global options passed to the `podman runlabel` command, allowing custom storage configurations to be passed into containers run with `runlabel` ([#2399](https://github.com/containers/libpod/issues/2399))
+- The `podman play kube` command now allows `File` and `FileOrCreate` volumes
+- The `podman pod prune` command was added to prune unused pods
+- Added the `podman system migrate` command to migrate containers using older configurations to allow their use by newer Libpod versions ([#2935](https://github.com/containers/libpod/issues/2935))
+- Podman containers now forward proxy-related environment variables from the host into the container with the `--http-proxy` flag (enabled by default)
+- Read-only Podman containers can now create tmpfs filesystems on `/tmp`, `/var/tmp`, and `/run` with the `--read-only-tmpfs` flag (enabled by default)
+- The `podman init` command was added, performing all container pre-start tasks without starting the container to allow pre-run debugging
+
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: I'll just once again note my dislike for no ending periods and will quietly move along.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+### Bugfixes
+- Fixed a bug where `podman cp` would not copy folders ([#2836](https://github.com/containers/libpod/issues/2836))
+- Fixed a bug where Podman would panic when the Varlink API attempted too pull a non-existent image ([#2860](https://github.com/containers/libpod/issues/2860))
+- Fixed a bug where `podman rmi` sometimes did not produce an event when images were deleted
+- Fixed a bug where Podman would panic would panic when the Varlink API passed improperly-formatted options when attempting to build ([#2869](https://github.com/containers/libpod/issues/2869))
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: s/would panic would panic/would panic/

**Comment by mheon** [comment]
Pushed updated version with fix for Tom's comment, plus updated the release notes for Buildah to indicate we grabbed 1.8.2

**Pull request merged**